src/backend/core/tests/team_accesses/test_api_team_accesses_list.py

```diff
@@ -203,3 +203,82 @@ def test_api_team_accesses_list_authenticated_constant_numqueries(
 
     assert response.status_code == 200
     assert response.json()["count"] == 21
+
+
+def test_api_team_accesses_list_authenticated_ordering():
+    """Team accesses can be ordered by "role"."""
+
+    user = factories.UserFactory()
+    factories.IdentityFactory(user=user, is_main=True)
+
+    team = factories.TeamFactory()
+    models.TeamAccess.objects.create(team=team, user=user)
+
+    # create 20 new team members
+    for _ in range(20):
+        extra_user = factories.IdentityFactory(is_main=True).user
+        factories.TeamAccessFactory(team=team, user=extra_user)
+
+    client = APIClient()
+    client.force_login(user)
+
+    response = client.get(
+        f"/api/v1.0/teams/{team.id!s}/accesses/?ordering=role",
+    )
+    assert response.status_code == 200
```

Comment:
Afterthought, and @sampaccoud comment on one of your PR, we might prefer constant value. I'll still fix up my PR and then refactor all constant values.

Reply:
Actually, we are not using DRF's status codes in this file. Afterthough, I reconsidered Sam's comment on "you should not use constant value in your test" I think we should replace DRFS's status code constants by hardcoded value to keep our tests dumb as possible.

Reply:
You know my position but the other day I also validated the commit proposing to add constants :laughing:  so... :man_shrugging: should we call it nitpicking?

Reply:
It might .... but I removed all these hard-coded values in my latest commit. I explain why in the commit's body. I think we took the wrong decision, I made a mistake, and I prefer fixing it rn. nothing that important btw

Reply:
I still think status codes are clearer. Reading the name and not having to remember what this number means alleviates our brain charges, if even of 1%. It doesn't matter most of the time but sometimes it saves us (or at least me) some time.

Should almost count for accessibility for ADHD, I guess.

Reply:
Regarding your comments on the commit body : 
- I don't think people used to the codes are actually disturbed by a constant being "<the code>_<the explanation"
- as for freezegun, we add the necessary packages if we decide they bring something to our test suite. Additionnaly, DRF is a dependency we already have.

Reply:
That's not the point, people used to use literals, will continue to use them. When reviewing their code, we'll nitpick those literals, they will have to fix up their code, which is totally functional, the reviewer will have to review again the code, and everyone will lose time for no added value. The code was functional with a literal, it will be with the constant.

Reply:
When something doesn't matter, most of the time, we should focus on what's bring value

Reply:
freezgun is a bit different, we need it to run/write our tests without create a monstrosity of flakiness. It will also improve code readability, having a context stating the date the object is created, is much more explicit than a sleep(1)

Reply:
I will revert my commit and align my code to maintain the code accessible